daemon/call.c

```diff
@@ -2595,7 +2597,12 @@ int monologue_offer_answer(struct call_monologue *other_ml, GQueue *streams,
 
 		/* get that many ports for each side, and one packet stream for each port, then
 		 * assign the ports to the streams */
-		em = __get_endpoint_map(media, num_ports, &sp->rtp_endpoint);
+		if (!bridge_count) {
+			em = __get_endpoint_map(media, num_ports, &sp->rtp_endpoint, 0);
+		} else {
+			unsigned int *bridge_port = (unsigned int *)g_queue_pop_head(&call->rtp_bridge_ports);
```

Comment:
These are supposed to be dynamically allocated "int"s? You can skip this detour by casting the int directly to a pointer and then back, which also eliminates possible memory leaks. See GINT_TO_POINTER() and GPOINTER_TO_INT() or just typecast it yourself.